README.md

```diff
@@ -158,6 +158,9 @@ the Cloud Spanner Java client.
 
 ## OpenCensus Metrics
```

Comment:
Will we be adding instructions for how to use `OpenTelemetry` in a separate PR?

Reply:
Yes, I was planing to create a separate PR for documentation. We would be writing a blog as well, the link will also need to be added here. We might also need to add a sample for the same. 
So was planning to cover it in a separate PR.

Reply:
ack